pages-activities/activities-learning.html

```diff
@@ -174,4 +174,23 @@ <h4>
             </div>
         </div>
     </div>
+    <div class="col-lg-4 col-md-6 d-flex align-items-stretch">
+        <div class="row">
+            <div class="card">
+                <h4>
+                    <span class="fas fa-frog"></span>
+                    <a href="https://flexboxfroggy.com/" target="_blank">FlexBox Froggy</a>
+                    <span class="badge badge-warning">Easy</span>
```

Comment:
Easy activities are usually highlighted green, which uses the 'badge-success' class.

Reply:
Done